Adiciona post Mario Sergio (#132)
okfn-brasil/embaixadoras: _posts/2023-05-23-saida-mario-sergio.md

---

> title: Fundador da rede de Embaixadoras, Mário Sérgio Queiroz passa a integrar equipe de participação digital no governo federal

Reviewer: ```suggestion
title: Mário Sérgio Queiroz passa a integrar equipe de participação digital no governo federal
```
Solicitando a retirada do trecho do título pra acompanhar as outras mudanças no texto

categories:
- Notícias
- Dados Abertos
- Governo Federal
author_staff_member: rebeca_almeida
show_comments: true
---
*Ao tomar posse do cargo de Coordenador-Geral, ele se desliga do Conselho de Governança da Open Knowledge Brasil*

> ![Perfil Étnico](https://embaixadoras.ok.org.br/images/team/badged_team/mario_sergio.jpg)

Reviewer: ```suggestion
![Perfil Mario Sergio](https://embaixadoras.ok.org.br/images/team/badged_team/mario_sergio.jpg)
```

> O conselheiro de Governança e membro da Rede de Embaixadoras de Inovação Cívica da OKBR, Mário Sérgio Queiroz, foi nomeado para o cargo de Coordenador-Geral de Participação Digital na Secretaria Nacional de Participação Social, órgão vinculado à Secretaria-Geral da Presidência da República. Ele atuará na equipe de Laila Bellix, também ex-integrante do conselho da OKBR, que assumiu a diretoria de Participação Digital em março deste ano. 

Reviewer: ```suggestion
O conselheiro de Governança da OKBR e membro da Rede de Embaixadoras de Inovação Cívica, Mário Sérgio Queiroz, foi nomeado para o cargo de Coordenador-Geral de Participação Digital na Secretaria Nacional de Participação Social, órgão vinculado à Secretaria-Geral da Presidência da República. Ele atuará na equipe de Laila Bellix, também ex-integrante do conselho da OKBR, que assumiu a diretoria de Participação Digital em março deste ano. 
```
Adicionando pra não parecer que ele é conselheiro da rede de embaixadoras

> O conselheiro de Governança e fundador da Rede de Embaixadoras de Inovação Cívica da OKBR, Mário Sérgio Queiroz, foi nomeado para o cargo de Coordenador-Geral de Participação Digital na Secretaria Nacional de Participação Social, órgão vinculado à Secretaria-Geral da Presidência da República. Ele atuará na equipe de Laila Bellix, também ex-integrante do conselho da OKBR, que assumiu a diretoria de Participação Digital em março deste ano. 

Reviewer 2: ```suggestion
O conselheiro de Governança e membro da Rede de Embaixadoras de Inovação Cívica da OKBR, Mário Sérgio Queiroz, foi nomeado para o cargo de Coordenador-Geral de Participação Digital na Secretaria Nacional de Participação Social, órgão vinculado à Secretaria-Geral da Presidência da República. Ele atuará na equipe de Laila Bellix, também ex-integrante do conselho da OKBR, que assumiu a diretoria de Participação Digital em março deste ano. 
```

Reviewer: Acho que precisa ser "O conselheiro de Governança da OKBR e membro da Rede de Embaixadoras..." 
Se não adicionar o OKBR, fica parecendo que ele é conselheiro da rede de embaixadoras.

Entre outras atribuições, a Secretaria é responsável por articular ações e estruturas de participação nos órgãos da administração direta e indireta do governo federal, além de coordenar o calendário das conferências nacionais. Há uma semana (11/5), o órgão lançou a plataforma Brasil Participativo, construída a partir do software livre Decidim.

> Mário Sérgio Queiroz é cientista da computação, já esteve à frente da área de Inovação Cívica da OKBR. Durante sua atuação, idealizou e fundou a rede de Embaixadoras, com o objetivo de manter próximas as pessoas interessadas em atuar ativamente no cenário de inovação cívica brasileiro. 

Reviewer 2: Não sei se a escrita da sugestão ficou boa mas odemos acrescentar a atuação na construção da comunidade de código e no querido diário (cujo lançamento foi quase todo organizado na gestão dele também)
```suggestion
Mário Sérgio Queiroz é cientista da computação, já esteve à frente da área de Inovação Cívica da OKBR. Durante sua atuação, a rede de Embaixadoras foi fundada, com o objetivo de manter próximas as pessoas interessadas em atuar ativamente no cenário de inovação cívica brasileiro. Os projetos de código aberto, como o Querido Diário, também tiveram seu desenvolvimento alavancado e estimularam a criação da comunidade de código aberto centralizada no Discord, que hoje conta com mais de 700 pessoas.
```

Ele também foi presidente da Associação Python Brasil. Antes de assumir a posição no governo federal, atuava no Jusbrasil, no relacionamento com desenvolvedores. Após sair da OKBR ele passou a integrar a Rede de Embaixadoras e o corpo de associados da organização, como voluntário. 

> Queiroz fazia parte do Conselho de Governança da Open Knowledge Brasil desde setembro de 2022, quando foi eleito em Assembleia Geral da organização. Ao assumir o posto no governo federal, ele se desliga do Conselho..

Reviewer 2: ```suggestion
Mário Sérgio fazia parte do Conselho de Governança da Open Knowledge Brasil desde setembro de 2022, quando foi eleito em Assembleia Geral da organização. Ao assumir o posto no governo federal, ele se desliga do Conselho.
```
